go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -172,6 +215,13 @@ func Init() (err error) {
 	}
 	log.Info("SBI Initialized")
 
+	distancePeriodicTicker = time.NewTicker(time.Second)
```

Comment:
Is the periodic timer configurable via the API or does it always have to be 1 second.

Reply:
the idea is to have a main 1s timer and that each subscription has a counter the gets decremented each time this timer expires... The reason is that every subscription can have its own frequency (in seconds)... so instead of having N timers... i have 1 timer and N counters that i check .. i believe it is better this way because it can end up being really a lot of timer resources otherwise.

Reply:
okay to leave as is, although it would be fine for each subscription to have its own timer.